furiosa/models/data/enf_generator.py

```diff
@@ -2,37 +2,41 @@
 from multiprocessing import Pool
 import os
 from pathlib import Path
+import re
+import shlex
+import subprocess as sp
 import tempfile
 from typing import Tuple
 
 import onnx
 import yaml
 
 from furiosa.quantizer import ModelEditor, TensorType, get_pure_input_names, quantize
-from furiosa.tools.compiler.api import VersionInfo, compile
 
 COMPILER_CONFIG = {"lower_tabulated_dequantize": True}
 
+compiler_info = sp.run(
+    ['furiosa-compiler', '--version'], capture_output=True, text=True, check=True
+).stdout
+pattern = r"frontend:\n- version: (.+?)\n- revision: (.+?)\b"
```

Comment:
frontend version is furiosa-rt version, and we should use backend version based on npu-tools compiler (== npu-ir). Also, the revision doesn't seem to be correct. You may need to check what SDK versions you used to generate this ENF.

```
furiosa-compiler --version
furiosa-compiler v0.10.1 (8b00177dc 2023-10-12T07:44:15Z)
backend:
- version: 0.10.1
- revision: 8b00177dc
- built_at: 2023-10-12T07:44:15Z
frontend:
- version: 0.10.2
- revision: a45bb1a0b
- built_at: 2023-10-12T07:54:21Z
```

Reply:
This PR's version uses furiosa-compiler version 0.10.0 from [Makefile](https://github.com/furiosa-ai/furiosa-models/blob/main/Makefile#L4C1-L4C30).
I'll make another PR to bump up toolchain version

Reply:
Added 0.10.2 support in 5cd04ec :)